ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
@dlech, is it possible for this function to be useful?

1. If the motor is running, and we call the function, it will return immediately.
2. If the motor is not running, is there any possibility for the function to actually return? (except for multithreaded operations).

I only see it used [here](https://github.com/rhempel/ev3dev-lang-python/pull/118/files#diff-99966894a0f2bcb1a5bc3f35219da892R39), right after the call to `run-to-rel-position`. Is it possible for the state to not contain `'running'` after the run command was sent to the motor?